source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -121,3 +121,50 @@
 		}
 	}
 }
+
+.is-style-features {
+	--wporg--style--feature--offset: 0.2em;
+	padding: 0;
+	list-style: none;
+	font-size: var(--wp--preset--font-size--heading-1);
+	font-weight: 200;
+	line-height: 1.1;
+
+	li {
+		overflow: hidden;
+		box-sizing: border-box;
+		height: 1.1em;
+		padding-top: var(--wporg--style--feature--offset);
+		padding-left: var(--wp--preset--spacing--50);
+		padding-right: var(--wp--preset--spacing--50);
+		color: #6682ff; // Hardcoded color for better contrast.
+		border-bottom: 1px solid var(--wp--preset--color--blueberry-3);
+		transition: all 0.2s ease-in-out;
+
+		a {
+			text-decoration: none;
```

Comment:
Minor, but if I add a link this underline still shows in the editor on hover

![Screen Shot 2022-08-04 at 4 34 55 PM](https://user-images.githubusercontent.com/1017872/182764100-35c19f4e-ee64-48ea-97f0-852b21d00ff0.jpg)

Reply:
Oh yep, it's not as visible for me in FF. Thanks for catching that.
<img width="300" alt="Screen Shot 2022-08-04 at 11 20 11 AM" src="https://user-images.githubusercontent.com/541093/182884762-939dbd4a-5e89-454e-a9c6-1ca733e70993.png">

Reply:
Looks like this is fixed with the link fix that came out in GB 13.8, once I ran `composer update` to get updated, no more underline:

<img width="270" alt="Screen Shot 2022-08-04 at 11 40 04 AM" src="https://user-images.githubusercontent.com/541093/182888746-f234a579-ab43-4235-bba2-bf612b8c6e78.png">